src/OSPSuite.Core/ICoreUserSettings.cs

```diff
@@ -1,20 +1,26 @@
 ﻿namespace OSPSuite.Core
 {
-   public interface ICoreUserSettings
-   {
-      /// <summary>
-      ///    Specifies how many core can be use by the application when performing parallel computations
-      /// </summary>
-      int MaximumNumberOfCoresToUse { get; set; }
+    public interface ICoreUserSettings
+    {
+        /// <summary>
+        ///    Specifies how many cores can be used by the application when performing parallel computations
+        /// </summary>
+        int MaximumNumberOfCoresToUse { get; set; }
 
-      /// <summary>
-      ///    Number of bin per plot
-      /// </summary>
-      int NumberOfBins { get; set; }
+        /// <summary>
+        ///    Number of bins per plot
+        /// </summary>
+        int NumberOfBins { get; set; }
 
-      /// <summary>
-      /// Number of individuals per bin
-      /// </summary>
-      int NumberOfIndividualsPerBin { get; set; }
-   }
+        /// <summary>
+        /// Number of individuals per bin
+        /// </summary>
+        int NumberOfIndividualsPerBin { get; set; }
+        
+        /// <summary>
+        /// Suppresses the warning when the user removes an ObservedData-entry from a simulation
+        /// </summary>
+        /// <remarks>This property is purposely not included in the serializer so the suppression works for the duration of an application-run.</remarks>
+        bool SuppressWarningOnRemovingObservedDataEntryFromSimulation { get; set; }
```

Comment:
This feels wrong to me. I can't really explain why but this is really granular to b in CoreUserSettings
Also it really does not scale. If we have more of those settings we ned to create one for all possible use cases. 
Lastly, this feels like it will be serialized. Bassed on the issue , it says something about per session. Or I did not understand what was meant

Reply:
Peter and I were discussing about where to put the flag. It's not serialized right now but I bet someone could make the mistake of adding it if they found this and thought it was left out by accident.

Peter originally had it as a static flag in the task. What do you think about leaving it there? With comment about why its static.

Reply:
Maybe a good use case to start a section in the dev docs, explaining what belongs where? E.g. a short description of what goes into `CoreUserSettings` und what should not.

Reply:
@PavelBal do you think there are other nag boxes that could use a flag like this? I imagine there are, and if so, then we should plan for a kind of session limited user settings object. 

In this case, I'd put a collection of dialog box Id's in a collection and that collection could be checked before the dialog is shown.